FalmouthFalcons/iteration-3
# Jim f ticket added search and styling

Description:
Added search and styling 

Number of Fixes:

Related Ticket(s):
#7 

Problem to Solve:
N/A

Proposed Changes:
None

Expected Behavior:
Search for products to buy

Steps to Test Solution
1) git pull origin master
2) git fetch
3) git checkout jim-f-ticket 


Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
